diff --git a/internal/store/storetest/storetest.go b/internal/store/storetest/storetest.go
index e533446a..d444f72b 100644
--- a/internal/store/storetest/storetest.go
+++ b/internal/store/storetest/storetest.go
@@ -265,6 +265,7 @@ func Dial(t testing.TB, recvLimit int, opts ...grpc.DialOption) *qdrant.Client {
 	if err != nil {
 		t.Fatalf("client: %v", err)
 	}
+	_, _ = c.Upsert(context.Background(), &qdrant.UpsertPoints{})
 	return c
 }
 
