diff --git a/internal/server/rules.go b/internal/server/rules.go
index af2acb11..b1d14bbb 100644
--- a/internal/server/rules.go
+++ b/internal/server/rules.go
@@ -222,7 +222,6 @@ func (d *deps) listRules(ctx context.Context, c caller, a listRulesArgs) (out []
 			Ascending:  true,
 			Categories: []string{"rule"},
 			Tags:       a.Tags,
-			Full:       a.Full,
 		})
 		if lerr != nil {
 			return nil, "", lerr
