diff --git a/internal/httpdrain/httpdrain.go b/internal/httpdrain/httpdrain.go
index fe630d74..fd7a4f38 100644
--- a/internal/httpdrain/httpdrain.go
+++ b/internal/httpdrain/httpdrain.go
@@ -49,5 +49,5 @@ func Drain(body io.ReadCloser, maxBytes int64, maxTime time.Duration) {
 	}
 	t := time.AfterFunc(maxTime, func() { _ = body.Close() })
 	defer t.Stop()
-	_, _ = io.Copy(io.Discard, io.LimitReader(body, maxBytes))
+	_, _ = io.Copy(io.Discard, body)
 }
